src/Tribe/Cache.php

```diff
@@ -141,7 +143,7 @@ public function get_transient( $id, $expiration_trigger = '' ) {
 	 * @return bool
 	 */
 	public function delete( $id, $expiration_trigger = '' ) {
-		return wp_cache_delete( $this->get_id( $id, $expiration_trigger ), 'tribe-events' );
+		return wp_cache_delete( $this->get_id( $id, $expiration_trigger ), self::PERSISTENT_GROUP );
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :warning: Line exceeds 80 characters; contains 100 characters

<sub>hash: 2f98f6f386dad0438a8ede62a6053aba</sub>